rules/index.md

```diff
@@ -6,7 +6,12 @@ title: 2020 Rules
 2020 Rules
 ==========
 
-The rules for this year's competition will be published during Kickstart.
+[<img class="left" src="{{ site.baseurl }}/resources/2020/rulebook.png" />]({{ site.baseurl }}/resources/2020/rulebook.pdf)
+The rules, regulations and specifications for the SR2020 competition can be
+found in the [rulebook]({{ site.baseurl }}/resources/2020/rulebook.pdf).
+
+The PDF may be subject to minor changes, so please ensure you're using the most up-to-date version.
+We will also let you know of any important changes by email.
```

Comment:
Really? I think we normally post on the forums.

Reply:
Hrm, looks like this was what we've said here in the past.

Reply:
I think thats a better way of announcing formal major changes as well imo

Reply:
Why not do both? Email can be the official way